### ISSUE TITLE
window.scrollX and window.scrollY not supported in IE

### ISSUE THREAD

**Issue by luke-venn-ft** [Financial-Times/o-tabs#22]
When selecting a tab in IE the page is scrolling back to the top. This is being caused when we try and reset the scroll after giving focus to the panel for screen readers due to the fact that IE doesn't have the scrollX and scrollY properties so should fallback to pageXOffset and pageYOffset.

**Comment by AlbertoElias** [comment]
Fixed in #23
